### PR TITLE
[#5919] Platform: change default storage option for AWS provider to 1x250GB

### PR DESCRIPTION
**Pull request by jitendra-12113**
Heading:
[5919][Platform] change default storage option for AWS provider to 1x250GB
Description:
We wanted to show default volume count to 1 on the UI for all the instance types starting with "c5./c4."
Testing:
When we select aws provider while creating universe, volume count was showing more than one for some instances types starting with ""c5./c4.". Now with this fix it will show only one.
Added Unit test as well.

### CODE REVIEW

File: managed/src/main/java/com/yugabyte/yw/cloud/AWSInitializer.java [hunk 1]
```diff
@@ -516,8 +518,7 @@ private void storeInstanceTypeInfoToDB() {
           LOG.error(msg);
           throw new UnsupportedOperationException(msg);
         } else {
-          // TODO: hardcode me not?
-          volumeCount = 2;
+          volumeCount = YB_VOLUME_INFO_VOLUME_COUNT;
```
Comment (SergeyPotachev) [inline managed/src/main/java/com/yugabyte/yw/cloud/AWSInitializer.java:521]: It looks like you forgot to read the value from the config file.

**Review by SergeyPotachev — CHANGES_REQUESTED** [review]

File: managed/src/main/java/com/yugabyte/yw/cloud/AWSInitializer.java (hunk 1 above)
Comment (sb-yb) [reply managed/src/main/java/com/yugabyte/yw/cloud/AWSInitializer.java:521]: FYI check `TaskGarbageCollector` for good example of how to inject config and use it. Other places are using deprecated code and that will be fixed soon.

**Review by sb-yb — CHANGES_REQUESTED** [review]
@jitendra-12113 Thanks for the change!
Like I commented on thread. I feel configurable way to do this is:
- AWS initializer sets the default to -1.
- InstanceTypes.java will check if the value in database is -1 use the default from configuration.

That way in future one can change configuration and the change takes effect without database migration. 
(You will  still need db migration `one time` to fix existing defaults and set it to -1)

IMHO if we do not do it this way then making it configurable has no value since default gets persisted in the db.
Let me know if this is clear or you want to discuss on VC. cc: @Arnav15

File: managed/src/main/java/com/yugabyte/yw/cloud/AWSInitializer.java (hunk 1 above)
Comment (jitendra-12113) [reply managed/src/main/java/com/yugabyte/yw/cloud/AWSInitializer.java]: Oops! My bad! Updated the patch. 

@sb-yb  I am not sure on how are we going to migrate DB for older instance types, need to check. Let's connect whenever your time permits to bring this to closure. Thanks!

**Pull request closed without merging**